README.md

```diff
@@ -121,17 +98,18 @@ To install HDP Search, follow the [HDP Search instructions](https://docs.hortonw
     ambari-server restart
     ```
 
-### 3. Setup DSX Local
+Please check the Solr is up and running, and the version is at 6.6.2. You can go the Ambari UI, go to Solr session, click the `Quick Links`, then click the `Solr UI`, it will go Solr's instance, it will show the versions. 
 
-This code pattern was tested using [DSX Desktop](https://www.ibm.com/products/data-science-experience), a lightweight version of DSX Local intended for standalone use and optimized for local development. For production deployments it is recommended to use DSX Local with a three node configuration. For information on how to do that, see the [DSX Install Docs](https://content-dsxlocal.mybluemix.net/docs/content/local/welcome.html).
+### 3. Setup DSX Desktop 
 
-Additionally, DSX Local provides documentation on how to set up HDP to work with DSX! Check out [Set up an HDP cluster for DSX](https://content-dsxlocal.mybluemix.net/docs/content/local/hdp.html) for more details.
+This code pattern was tested using [DSX Desktop - MacOS](https://www.ibm.com/products/data-science-experience), a lightweight version of DSX Local intended for standalone use and optimized for local development. For information on how to do that, see the [DSX Desktop Install Docs](https://content-dsxdesktop.mybluemix.net/docs/content/desktop/install.html). 
```

Comment:
MacOS is the link name ? Perhaps we can remove it ?

Reply:
Yeah.. coz.. if we are going to only support on Mac.. it would be very restrictive. I am suggesting we claim support for Mac and Linux.. But let some one else verify.. we will fix as needed.

Reply:
ok